specification/cognitiveservices/data-plane/WebSearch/readme.typescript.md

```diff
@@ -7,7 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   package-name: "@azure/cognitiveservices-websearch"
   output-folder: "$(typescript-sdks-folder)/packages/@azure/cognitiveservices-websearch"
-  override-client-name: WebSearchAPIClient
```

Comment:
The reason they were added because the title field of the swagger spec did not have a nice name. This would make sure that a relevant name is provided. By removing this setting, would the newly generated client still have the same name?

Reply:
They have a different name, but the cognitiveservices team specifically asked us to remove these properties.